spatialdata/_core/_transform_elements.py

```diff
@@ -246,3 +248,77 @@ def _(data: AnnData, transformation: BaseTransformation) -> AnnData:
 
     ShapesModel.validate(transformed_adata)
     return transformed_adata
+
+
+def get_transformation_between_landmarks(
+    references_coords: AnnData,
+    moving_coords: AnnData,
+) -> Affine:
```

Comment:
this should be documented. Also, wouldn't this go under some `utils` module?

Reply:
Added a docstring. Kevin and I propose to place the method in `spatialdata.transform`, see here: https://github.com/scverse/spatialdata/issues/103